src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/ReflectionJsonTypeInfoOfT.cs

```diff
@@ -43,13 +45,16 @@ internal ReflectionJsonTypeInfo(JsonConverter converter, JsonSerializerOptions o
 
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
                 Justification = "Constructor is marked as RequiresUnreferencedCode")]
```

Comment:
```suggestion
                Justification = "The ctor is marked as RequiresUnreferencedCode.")]
```
Nit: use wording that is used everywhere else